packages/php-wasm/node/src/lib/xdebug/with-xdebug.ts

```diff
@@ -0,0 +1,78 @@
+import type {
+	EmscriptenOptions,
+	PHPRuntime,
+	SupportedPHPVersion,
+} from '@php-wasm/universal';
+import { LatestSupportedPHPVersion } from '@php-wasm/universal';
+import { jspi } from 'wasm-feature-detect';
+import { fullyQualifiedPHPVersionDirectory } from './supported-php-versions';
+import { dirname } from 'path';
+import fs from 'fs';
+
+export async function withXdebug(
+	version: SupportedPHPVersion = LatestSupportedPHPVersion,
+	options: EmscriptenOptions
+): Promise<EmscriptenOptions> {
+	if (await jspi()) {
```

Comment:
Let's be explicit about errors. If withXdebug is called in an environment where xdebug is unsupported, let's bale out. Otherwise the user may be confused "but I enabled it, why doesn't it work?"

```suggestion
	if (!await jspi()) {
		throw new Error();
	}
```

Reply:
Done.